### PR TITLE
XCM: Replace council XCM origin with general admin

### PR DESCRIPTION
**Pull request by KiChjang**
This PR removes the ability to convert a Gov1 council origin to a corresponding `Plurality` `MultiLocation` value. In its place, the `GeneralAdmin` origin can be converted to a `Plurality` with the `BodyId` of `Adminstration`.

### CODE REVIEW

**Review by liamaharon — APPROVED** [review]

**Review by joepetrowski — APPROVED** [review]
I don't think General Admin is _necessary_ (i.e. we don't depend on it in remote locations like we do for [Staking Admin](https://github.com/paritytech/cumulus/blob/ce5ed1a/parachains/runtimes/assets/asset-hub-polkadot/src/lib.rs#L622)), but it seems safe to allow and reasonable that we would use it in remote locations.

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/3405473

**Review by bkchr — APPROVED** [review]

File: runtime/polkadot/src/xcm_config.rs [hunk 1]
```diff
@@ -350,7 +350,8 @@ impl xcm_executor::Config for XcmConfig {
 }
 
 parameter_types! {
-	pub const CouncilBodyId: BodyId = BodyId::Executive;
+	// GeneralAdmion pluralistic body.
```
Comment (ordian) [inline runtime/polkadot/src/xcm_config.rs:353]:
```suggestion
	// `GeneralAdmin` pluralistic body.
```

**Review by ordian — APPROVED** [review]

**Comment by liamaharon** [comment]
bot rebase

**Comment by liamaharon** [comment]
bot fmt

**Comment by KiChjang** [comment]
bot merge

**Pull request merged**